project/plugins.sbt

```diff
@@ -1,11 +1,13 @@
 addSbtPlugin("com.dwijnand" % "sbt-dynver" % "3.0.0")
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.2")
+// https://scalameta.org/scalafmt/#Changelog
+addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.0.0")
 addSbtPlugin("se.marcuslonnberg" % "sbt-docker" % "1.5.0")
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.3.2")
-addSbtPlugin("com.lightbend" % "sbt-whitesource"  % "0.1.10")
+addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.10")
+// https://github.com/ThoughtWorksInc/sbt-api-mappings/releases
```

Comment:
Why the comment to the releases page? Is it to easily track new versions? For the purpose I use `https://github.com/rtimush/sbt-updates` which I have in my global sbt config and it works great.